Update styling with sx prop in fallback.js

SAUMILDHANKAR opened the pull request:
- Closes #1730 
- Removed makeStyles.
- Removed unused figure tag since there is no image.
- Removed html div tags; imported and used relevant mui components.
- Adjusted fontSize and lineHeight to match styling in h2 variant.
- Screenshots added below to make sure no visible changes to the styling.

Web screenshot before
<img src="https://github.com/hackforla/food-oasis/assets/843538/ce11b313-32f1-4d3c-940a-051a6718f7e6.jpeg" width="90%" height="60%">
Web screenshot after
<img src="https://github.com/hackforla/food-oasis/assets/843538/1e8e1631-de82-4f5b-90a6-9956e547ec16.jpeg" width="90%" height="60%">

Mobile screenshot before
<img src="https://github.com/hackforla/food-oasis/assets/843538/c96840e5-ba77-4acf-bb83-6dcf4a23ddc6.png" width="40%" height="80%">
Mobile screenshot after
<img src="https://github.com/hackforla/food-oasis/assets/843538/a622bafe-a417-4cf8-8412-50eca448acd1.png" width="40%" height="80%">

junjun107 reviewed (commented):
Does the outer <Box> do anything? If not, can we remove it?

SAUMILDHANKAR commented:
> Does the outer do anything? If not, can we remove it?

@junjun107 Thanks for your suggestion. I have removed the outer box and added width property to the box replacing main class styling. 

Simply removing the outer box was shrinking the whole thing but now it looks the same. Hope it is fine. Thank you.

junjun107 reviewed (approved):
LGTM
